assets/js/pages/Users/Users.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '@common/Button';
+import Table from '@common/Table';
+import PageHeader from '@common/PageHeader';
+import Modal from '@common/Modal';
+import Tooltip from '@common/Tooltip';
+import Banner from '@common/Banners/Banner';
+import { EOS_LOADING_ANIMATED } from 'eos-icons-react';
+
+const USER_CREATE_ROUTE = '/users/new';
+
+function Users({
+  handleDeleteUser = () => {},
+  navigate = () => {},
+  setModalOpen = () => {},
+  setDeleteUserId = () => {},
+  deleteUserId = 0,
+  modalOpen = false,
+  users = [],
+  loading = false,
+}) {
+  const usersTableConfig = {
+    pagination: true,
+    usePadding: false,
+    columns: [
+      {
+        title: 'Username',
+        key: 'username',
+        render: (content, item) => (
+          <Link
+            className="text-jungle-green-500 hover:opacity-75"
+            to={`/users/${item.id}/edit`}
+          >
+            {content}
+          </Link>
+        ),
+      },
+      {
+        title: 'Full Name',
+        key: 'fullname',
+      },
+      {
+        title: 'Email',
+        key: 'email',
+      },
+      {
+        title: 'Status',
+        key: 'enabled',
+      },
+      {
+        title: 'Created',
+        key: 'created',
+      },
+
+      {
+        title: 'Actions',
+        key: 'actions',
+        render: (content, item) => (
+          <>
+            {item.id !== 1 ? (
```

Comment:
I don't think we need this `if/else` case.
We should use the `disabled` prop for the button, and simply choose the styles based on the user id